iteration/iteration_test.go

```diff
@@ -432,3 +455,88 @@ func (s *TestIterationRepository) TestExistsIteration() {
 	})
 
 }
+
+func (s *TestIterationRepository) TestIsActive() {
+	t := s.T()
+	t.Run("user active is true", func(t *testing.T) {
+		// given
+		newSpace := space.Space{
+			Name:    testsupport.CreateRandomValidTestName("Space Exists"),
+			OwnerId: s.testIdentity.ID,
+		}
+		repoSpace := space.NewRepository(s.DB)
+		space, err := repoSpace.Create(context.Background(), &newSpace)
+		assert.Nil(t, err)
+
+		level0IterationName := "Top level iteration"
+		trueUserActive := true
+		i1 := iteration.Iteration{
+			Name:       level0IterationName,
+			SpaceID:    space.ID,
+			UserActive: &trueUserActive,
+		}
+		require.True(t, i1.IsActive())
+	})
+	t.Run("start date is nil", func(t *testing.T) {
+		// given
+		newSpace := space.Space{
+			Name:    testsupport.CreateRandomValidTestName("Space Exists"),
+			OwnerId: s.testIdentity.ID,
+		}
+		repoSpace := space.NewRepository(s.DB)
+		space, err := repoSpace.Create(context.Background(), &newSpace)
+		assert.Nil(t, err)
+
+		level0IterationName := "Top level iteration"
+		trueUserActive := false
```

Comment:
variable name `trueUserActive` has false value

Reply:
fixing it

Reply:
addressed it here 49ce9fc